go/vt/vtexplain/vtexplain.go

```diff
@@ -336,6 +341,26 @@ func ExplainsAsText(explains []*Explain) string {
 	return b.String()
 }
 
+func specialHandlingOfSavepoints(q *MysqlQuery) {
+	if strings.Contains(q.SQL, "savepoint") {
+		stmt, _ := sqlparser.Parse(q.SQL)
```

Comment:
we should not ignore the error, and we should use a checked type conversion so we don't panic if we got something other than a savepoint